src/helpers/runWorkflow.js

```diff
@@ -1,5 +1,5 @@
 const sequence = require("when/sequence");
-const Table = require("cli-table2");
+const Table = require("cli-table3");
```

Comment:
I checked, there is no `cli-table4` ha ha (yet)

Reply:
Hahaha. Yeah I thought it was funny that people just keep making a new project to keep supporting the previous library because they stopped keeping it updated. 😁